bft/engine.go

```diff
@@ -17,13 +16,20 @@ import (
 	"github.com/vechain/thor/v2/kv"
 	"github.com/vechain/thor/v2/muxdb"
 	"github.com/vechain/thor/v2/state"
+	"github.com/vechain/thor/v2/telemetry"
 	"github.com/vechain/thor/v2/thor"
+
+	lru "github.com/hashicorp/golang-lru"
 )
 
 const dataStoreName = "bft.engine"
 
 var finalizedKey = []byte("finalized")
 
+var (
+	metricsBlocksCommitted = telemetry.LazyLoadCounterVec("block_bft_committed_count", []string{"status"})
```

Comment:
Generally, I think there are two styles of naming:

+ `metricsXxxxx`
+ `xxxxxxCounterVec`

We need to reach a consensus in the team about it, which one do you prefer?

Reply:
`metricsXxxxx` are typically`metricsOfSomethingInParticular`, an instantiated instrument that takes metrics of a specific purpose.  For example,`metricsBlocksCommitted` doesn't need to specify if is a counter or a gauge in the name. I think it's ok to not specify the type of the instrument, bc in the end the underlying instrument can change.

`xxxxxxCounterVec` are typically `instrumentType` which allow instantiating a specific metric. This func needs a type description, but not an instance description.

Reply:
Ok, let's go with `metricsXxxxx`